core/storage/storage_models_test.py

```diff
@@ -35,8 +34,9 @@ def _get_model_module_names(self):
         # As models.NAMES is an enum, it cannot be iterated. So we use the
         # __dict__ property which can be iterated.
         for name in models.NAMES.__dict__:
-            if '__' not in name:
-                all_model_module_names.append(name)
+            if '__' in name:
```

Comment:
Why change this from what it was previously?

Reply:
Mistake, done.